### DISCUSSION TITLE
论文提交时未确定具体日期，可否支持答辩日期只到月份

### DISCUSSION THREAD

**Discussion by LeeCode518** [sjtug/SJTUThesis#866] [提问]

**Comment by LogCreative** [comment]
修改文档类代码为需要的格式即可

https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/texmf/tex/latex/sjtuthesis/sjtuthesis.cls#L388

https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/texmf/tex/latex/sjtuthesis/sjtuthesis.cls#L395


或者直接在导言区添加

```latex
\makeatletter
\renewcommand\sjtu@date@format@zh[3]{#1 年 \number#2 月}
\renewcommand\sjtu@date@format@en[3]{\sjtu@date@month@en{#2}, #1}
\makeatother
```
